lib/helper/Puppeteer.js

```diff
@@ -363,6 +363,7 @@ class Puppeteer extends Helper {
   /**
    * Checks that the active JavaScript popup, as created by `window.alert|window.confirm|window.prompt`, contains the
    * given string.
+   * @param {string} text value to check.
```

Comment:
Thank you!
Can you move this text to webapi docs? 
path: `docs/webapi`

Reply:
Okay, done.